### PR TITLE
Update aria properties and other docs tweaks

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Update ARIA property descriptions for consistency and to address https://github.com/blackbaud/skyux2-docs/issues/919, plus clean up a few other minor docs things I noticed while checking the component docs in this repo.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-aria-link-120220](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-aria-link-120220)

File: src/app/public/modules/autocomplete/autocomplete.component.ts [hunk 1]
```diff
@@ -70,7 +70,9 @@ export class SkyAutocompleteComponent
   //#region public_api
 
 /**
- * Identifies the element that defines a label for the text input.
+ * Specifies the HTML element ID (without the leading `#`) of the element that labels
+ * the autocomplete text input. This sets set the input's `aria-labelledby` attribute
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/autocomplete/autocomplete.component.ts:74]:
```suggestion
 * the autocomplete text input. This sets the input's `aria-labelledby` attribute
```

File: src/app/public/modules/lookup/lookup.component.ts [hunk 2]
```diff
@@ -71,14 +71,17 @@ export class SkyLookupComponent
...
+ * [for accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
  * If a label is visible on the screen, use the `ariaLabelledBy` property instead.
  */
   @Input()
   public ariaLabel: string;
 
 /**
- * Identifies the element that defines a label for the typeahead search input.
+ * Specifies the HTML element ID (without the leading `#`) of the element that labels
+ * the typeahead search input. This sets set the input's `aria-labelledby` attribute
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/lookup/lookup.component.ts:83]:
```suggestion
 * the typeahead search input. This sets the input's `aria-labelledby` attribute
```

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-aria-link-120220](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-aria-link-120220)

**Pull request merged**